Gymphony/src/app/core/components/navbar/navbar.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { UserService } from '../../services/user.service';
+import { User } from '../../interfaces/user';
+
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrl: './navbar.component.css'
+})
+export class NavbarComponent {
+  public user?: User;
```

Comment:
I recommend you to create variable like that:
public user$: Observable<User> = this.userService.user$;

This way you can subscribe on it in template by using async pipe

Advantage of this approach is that template subscription via sync pipe is destroyed automatically with the view